clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/IndexWebService.scala

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.clio.server.webservice
+
+import akka.http.scaladsl.server._
+import akka.http.scaladsl.server.Directives._
+import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
+  ClioDocument,
+  ElasticsearchIndex
+}
+import org.broadinstitute.clio.server.service.IndexService
+import org.broadinstitute.clio.transfer.model.TransferIndex
+
+abstract class IndexWebService[TI <: TransferIndex, D <: ClioDocument: ElasticsearchIndex](
+  val indexService: IndexService[TI, D]
+) extends JsonWebService {
+
+  import indexService.transferIndex.implicits._
+
+  val servicePathPrefix: String
```

Comment:
We shouldn't need this; we can use [this](https://github.com/broadinstitute/clio/blob/develop/clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferIndex.scala#L17) instead.